examples/compare_coherency_methods.py

```diff
@@ -0,0 +1,593 @@
+#          Mohammad Orabe <orabe.mhd@gmail.com>
+# License: BSD (3-clause)
+
+# %%
+import numpy as np
+from matplotlib import pyplot as plt
+
+import mne
+from mne_connectivity import seed_target_indices, spectral_connectivity_epochs
+
+###############################################################################
+# An introduction to coherency-based connectivity methods
+# -------------------------------------------------------
+#
+# MNE-Connectivity supports several methods based on coherency. These are:
+#
+# - coherency (Cohy)
+# - coherence (Coh; absolute coherency)
+# - imaginary part of coherency (ImCoh)
+# - canonical coherency (CaCoh)
+# - maximised imaginary part of coherency (MIC)
+# - multivariate interaction measure (MIM)
+#
+# |
+#
+# All of these methods centre on Cohy, a complex-valued estimate of the
+# correlation between signals in the frequency domain. It is an undirected
+# measure of connectivity, being invariant to the direction of information flow
+# between signals.
+#
+# The common approach for handling these complex-valued coherency scores is to
+# either take their absolute values (Coh) or their imaginary values (ImCoh
+# :footcite:`NolteEtAl2004`).
+#
+# In addition to these traditional bivariate connectivity measures (i.e.
+# between two signals), advanced multivariate measures (i.e. between groups of
+# signals) have also been developed based on Cohy (CaCoh
+# :footcite:`VidaurreEtAl2019`; can take the absolute value for a multivariate
+# form of Coh) or ImCoh (MIC & MIM :footcite:`EwaldEtAl2012`).
+#
+# Despite their similarities, there are distinct scenarios in which these
+# different methods are most appropriate, as we will show in this example.
+
+###############################################################################
+# Zero and non-zero time-lag interactions
+# ---------------------------------------
+#
+# The key difference between Cohy/Coh and ImCoh is how information about zero
+# time-lag interactions is captured.
+#
+# We generally assume that communication within the brain involves some delay
+# in the flow of information (i.e. a non-zero time-lag). This reflects the time
+# taken for: the propagation of action potentials along axons; the release of
+# neurotransmitters from presynaptic terminals and binding to receptors on
+# postsynaptic terminals; etc...
+#
+# In contrast, interactions with no delay (i.e. a zero time-lag) are often
+# considered to reflect non-physiological activity, such as volume conduction
+# - the propagation of electrical activity through the brain's conductive
+# tissue from a single source to multiple electrodes simultaneously
+# :footcite:`NolteEtAl2004`. Such interactions therefore do not reflect
+# genuine, physiological communication between brain regions. Naturally,
+# having a method that can discard spurious zero time-lag connectivity
+# estimates is very desirable.
+#
+# **N.B.** Not all zero time-lag interactions are necessarily non-physiological
```

Comment:
What is N.B.?

Reply:
Just Latin for important point/take note. But can replace to avoid any confusion:
https://github.com/mne-tools/mne-connectivity/blob/8652da468637d12b5ae7f9c3ee580cd76a55b3d4/examples/compare_coherency_methods.py#L77-L78